src/commands/generate-odata-client.ts

```diff
@@ -43,6 +43,12 @@ export default class GenerateODataClient extends Command {
     sdkAfterVersionScript: toBooleanFlag(
       generatorOptionsSDK.sdkAfterVersionScript
     ),
+    versionInPackageJson: toStringFlag(
+      generatorOptionsSDK.versionInPackageJson!
```

Comment:
This is optional parameter, without default value, we should consider null check.

Reply:
the `generatorOptionsSDK` is an object instance just mapping the name of the variable `versionInPackageJson` to the options in `yargs` like short name, type etc. So this is always there. All options are described there and not null. The interface `GeneratorOptionsSDK` is then used at runtime and here some keys could be non optional.  

However, I found that with `-?` you can remove the modifier from the `keyof` so we can get rid of all the `!`.

Reply:
Maybe you can share the doc regarding `-?`